common/lib/container-definitions/package-lock.json

```diff
@@ -596,7 +596,7 @@
     },
     "@fluidframework/protocol-definitions": {
       "version": "0.1026.0-42695",
-      "resolved": "https://registry.npmjs.org/@fluidframework/protocol-definitions/-/protocol-definitions-0.1026.0-42695.tgz",
+      "resolved": "https://registry.npmjs.org/@fluidframework/protocol-definitions/-/protocol-definitions-0.1026.0-43898.tgz",
```

Comment:
this is wrong. the sha should change when the version does re-install in this package